experimental/IntersectionTheory/docs/src/examples.md

```diff
@@ -0,0 +1,20 @@
+```@meta
+CurrentModule = Oscar
+DocTestSetup = quote
+  using Oscar
+end
```

Comment:
In the future please use this (I'll not update it now to avoid merge conflicts):
```suggestion
DocTestSetup = Oscar.doctestsetup()
```